Samples~/Encryption/EncryptionSample.cs

```diff
@@ -0,0 +1,571 @@
+
+        frameTypeToCryptoOffset = GetFrameTypeToCryptoOffset(WebRTCSettings.UseVideoCodec?.mimeType);
+
+        pc1OnIceConnectionChange = state => { OnIceConnectionChange(_pc1, state); };
+        pc2OnIceConnectionChange = state => { OnIceConnectionChange(_pc2, state); };
+        pc1OnIceCandidate = candidate => { OnIceCandidate(_pc1, candidate); };
+        pc2OnIceCandidate = candidate => { OnIceCandidate(_pc2, candidate); };
+        pc2Ontrack = e =>
+        {
+            var receiver = e.Receiver;
+            pc1Receivers.Add(receiver);
+            SetUpReceiverTransform(receiver);
+            receiveStream.AddTrack(e.Track);
+        };
+        pc1OnNegotiationNeeded = () => { StartCoroutine(PeerNegotiationNeeded(_pc1)); };
+
+        receiveStream.OnAddTrack = e =>
+        {
+            if (e.Track is VideoStreamTrack track)
+            {
+                track.OnVideoReceived += tex =>
+                {
+                    receiveImage.texture = tex;
+                    receiveImage.color = Color.white;
+                };
+            }
+        };
+    }
+
+    private void OnChangedEncryptKeyInput(string data)
+    {
+        byte[] bytes = System.Text.Encoding.UTF8.GetBytes(data);
+
+        lock (encryptKeyInputLock)
+        {
+            if (encryptKeyArray.IsCreated)
+                encryptKeyArray.Dispose();
+            if(bytes.Length > 0)
+                encryptKeyArray = new NativeArray<byte>(bytes, Allocator.Persistent);
+        }
+    }
+
+    private void OnChangedDecryptKeyInput(string data)
+    {
+        byte[] bytes = System.Text.Encoding.UTF8.GetBytes(data);
+
+        lock (decryptKeyInputLock)
+        {
+            if (decryptKeyArray.IsCreated)
+                decryptKeyArray.Dispose();
+            if (bytes.Length > 0)
+                decryptKeyArray = new NativeArray<byte>(bytes, Allocator.Persistent);
+        }
+    }
+
+    private void OnDestroy()
+    {
+        if (encryptKeyArray.IsCreated)
+            encryptKeyArray.Dispose();
+        if (decryptKeyArray.IsCreated)
+            decryptKeyArray.Dispose();
+        if (encryptedDataArray.IsCreated)
+            encryptedDataArray.Dispose();
+        if (decryptedDataArray.IsCreated)
+            decryptedDataArray.Dispose();
+    }
+
+    private void SetUpSenderTransform(RTCRtpSender sender)
+    {
+        sender.Transform = new RTCRtpScriptTransform(TrackKind.Video, e => OnSenderTransform(sender.Transform, e));
+    }
+
+    private void SetUpReceiverTransform(RTCRtpReceiver receiver)
+    {
+        receiver.Transform = new RTCRtpScriptTransform(TrackKind.Video, e => OnReceiverTransform(receiver.Transform, e));
+    }
+
+    void OnSenderTransform(RTCRtpTransform transform, RTCTransformEvent e)
+    {
+        lock (encryptKeyInputLock)
+        {
+            if(encryptKeyArray.IsCreated)
+            {
+                if(e.Frame is not RTCEncodedVideoFrame frame)
```

Comment:
```suggestion
                if (!(e.Frame is RTCEncodedVideoFrame frame))
```
`is not` syntax does not support on Unity 2020.3.